MPF.Processors/Redumper.cs

```diff
@@ -192,11 +192,14 @@ public override void GenerateSubmissionInfo(SubmissionInfo info, string basePath
                         info.CommonDiscInfo.Region = ProcessingTool.GetXGDRegion(xmid.Model.RegionIdentifier);
                     }
 
-                    if (HashTool.GetStandardHashes($"{basePath}.dmi", out _, out string? dmi1Crc, out _, out _))
+                    string? dmi1Crc = HashTool.GetFileHash($"{basePath}.dmi", HashType.CRC32);
+                    if (dmi1Crc != null)
                         info.CommonDiscInfo!.CommentsSpecialFields![SiteCode.DMIHash] = dmi1Crc?.ToUpperInvariant() ?? string.Empty;
```

Comment:
Null coalesce can be removed from all of these now, unless you want to remove the newly-added `if` statements instead.